quilt/tools/command.py

```diff
@@ -581,6 +587,7 @@ def main():
     build_p.add_argument("package", type=str, help="Owner/Package Name")
     buildpath_group = build_p.add_mutually_exclusive_group(required=True)
     buildpath_group.add_argument("-d", "--directory", type=str, help="Source file directory")
```

Comment:
`-d` and `-g` now have the same help text, which is kind of weird. Would be nice to have more explanation.

Though I feel like there has to be a better way to organize the options. We now have:
- Generate the build file
- Build the package
- Both of the above

The most "logical" solution would be to have an argument for 1, another argument for 2, or both arguments if you want both. But of course it doesn't work well if we want 2 to be the default with no options.